changelog/README.md

```diff
@@ -1,10 +1,16 @@
 # Changelog
 
+## 4th September 2023
+
+* Deprecated `StartUtc` property in [Reservations (ver 2023-06-06)](../operations/reservations.md#get-all-reservations-ver-2023-06-06). Use more explicit `ScheduledStartUtc` and `ActualStartUtc` properties instead. No discontinuation date has been set for now.
+
 ## 31st August 2023
 
 * Enabled [Portfolio Access Tokens](../guidelines/multi-property.md) for the following operations:
   * [Get all message threads](../operations/messagethreads.md#get-all-message-threads)
   * [Get all messages](../operations/messages.md#get-all-messages)
+* Extended [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2023-06-06) response with `ScheduledStartUtc` and `ActualStartUtc`.
```

Comment:
I would prefer that the two entries in the Changelog appear together, as they are published at the same time in the documentation. It would also make it a bit easier to understand. For example:

* In [Get all reservations](../operations/reservations.md#get-all-reservations-ver-2023-06-06):
  * Clarified behavior of `StartUtc` property, which is either the reservation's scheduled time, or the actual customer check-in time, whichever is earlier
  * Added new properties `ScheduledStartUtc` and `ActualStartUtc`, intended ultimately to replace `StartUtc`
  * Deprecated `StartUtc`, but no discontinuation date has been set for now

Reply:
Thanks Mike, agreed! I've merged the entries under 5th September, I've only clarified the affected version and that the properties have been added to the response.